src/Core/Domain/CartRule/QueryResult/EditableCartRuleActions.php

```diff
@@ -28,9 +28,6 @@
 
 namespace PrestaShop\PrestaShop\Core\Domain\CartRule\QueryResult;
 
-use PrestaShop\PrestaShop\Core\Domain\Product\Combination\ValueObject\CombinationId;
-use PrestaShop\PrestaShop\Core\Domain\Product\ValueObject\ProductId;
-
 class EditableCartRuleActions
```

Comment:
```suggestion
class CartRuleAction
```

Reply:
```suggestion
class CartRuleActionForEditing
```

Reply:
I'm not entirely sure, but at least we should drop the `s` there's only one action no? (although it contains several options)